### ISSUE TITLE
The size of embedding dim

### ISSUE THREAD

**Issue by liu-zhenhua** [gupta-abhay/ViT#3]
This work is very interested and fascinating. I have a question :  the size of embedding size and how you decide it?
Look forward to the release of pretrained model.

**Comment by gupta-abhay** [comment]
embedding size is coming from the paper definition -- the hidden dim of the encoder and the embedding dim. are usually the same. 

regarding the pre-trained models, I'll take a call about it at some point. I have it tested for about 100 iterations, but running for 300 epochs on ImageNet-1K, requires serious juice. And unless I can find a sustainable way to do it, I will not pursue this effort.
